doc/neps/nep-template.rst

```diff
@@ -13,15 +13,27 @@ Abstract
 
 The abstract should be a short description of what the NEP will achieve.
 
+Motivation and Scope
+--------------------
+
+This section describes the need for the NEP.  It should describe the existing
```

Comment:
This is now duplicate with "detailed description". The first two sentences of that section should be replaced. I can have a go at it.

Reply:
@hameerabbasi @rgommers ping